cpp/include/cudf/io/parquet.hpp

```diff
@@ -510,6 +527,24 @@ class parquet_writer_options {
   {
     _column_chunks_file_path.assign(file_path);
   }
+
+  /**
+   * @brief Sets the maximum row group size, in bytes.
+   */
+  void set_row_group_size_bytes(size_t size_bytes)
+  {
+    CUDF_EXPECTS(size_bytes >= 512 << 10, "512KB is the minimum rowgorup size");
```

Comment:
Avoid shift for readability, and make this message sound like the other one.
```suggestion
    CUDF_EXPECTS(size_bytes >= 512 * 1024, "The maximum row group size must be at least 512KB.");
```